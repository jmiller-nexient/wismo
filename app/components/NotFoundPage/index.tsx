import * as React from 'react';

const NotFoundPage: React.FC = () => (
  <article>
    <h1>Page not found.</h1>
  </article>
);

export default NotFoundPage;
