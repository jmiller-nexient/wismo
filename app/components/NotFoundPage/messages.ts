import { defineMessages } from 'react-intl';

export const scope = 'wismo.components.not-found-page';

export default defineMessages({
  message: {
    id: `${scope}.message`,
  },
});
