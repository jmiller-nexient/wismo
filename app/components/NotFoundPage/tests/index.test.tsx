import React, { ReactElement } from 'react';
import renderer from 'react-test-renderer';

import NotFoundPage from '..';

describe('<App />', () => {
  it('should render correctly', () => {
    const tree: ReactElement = renderer.create(
      <NotFoundPage />,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
