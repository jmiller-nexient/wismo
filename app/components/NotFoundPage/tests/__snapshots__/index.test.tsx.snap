// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render correctly 1`] = `
<article>
  <h1>
    Page not found.
  </h1>
</article>
`;
